tests/utils/test_fx_utils.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+import operator
+from typing import Type
+
+import pytest
+import torch
+from torch import nn
+from torch.fx.graph_module import GraphModule
+
+from composer.utils.fx_utils import count_op_instances, replace_op, trace_with_fx
+
+
+class SimpleModel(nn.Module):
```

Comment:
That model is fx traceable but due to the use of sequential, it doesn't have the operations I want to replace. Changed the name to TestModel to avoid confusing it with SimpleModel.